src/main/webapp/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "Alpollo",
+  "name": "Alpollo",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    },
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
```

Comment:
Make sure you are using icons/logos with a valid license (better yet if you created your own)

Reply:
This was added by create-react-app (free for commercial use, https://github.com/facebook/react/), it's the part of the default project, so I think the icons are also free, but of course I'm going to do new ones

Reply:
Awesome. They seem to be free for commercial use as well (https://github.com/facebook/react/blob/master/LICENSE), so feel free to use them int this pull request and add the new icons only later

Reply:
Done in https://github.com/googleinterns/step217-2020/pull/5